_rules/text-contrast-afw4f7.md

```diff
@@ -176,19 +175,15 @@ This dark gray text has a contrast ratio of 12.6:1 on the white background in a
 
 #### Passed Example 10
 
-This text is part of a widget because it is a child of a `button` element. The text has the default
-browser button text color on the default browser button background color. By default, this is black text on a
-light gray background with a contrast ratio of 18.26:1
+This text has the [default user agent link text and background color](https://html.spec.whatwg.org/multipage/rendering.html#phrasing-content-3), of `#0000EE` and white. This results in a contrast ratio of 9.39:1.
```

Comment:
Wouldn't it be more unambiguous to change "white" to #FFF?

Reply:
🤔 Not sure. `white` is a keyword for colors, so it should be OK.
OTOH it seems that all previous examples were having explicit colors code in their `style`…

Reply:
I do not feel very strongly about it. Happy to approve it as I understand that "white" is an actual keyword. I just thought that it would be safer to go with a hex value.